index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Portfolio _one</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" crossorigin />
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header>
+        <ul>
+            <li><a class="myLogo font-style" href="">My Logo</a> </li>
+            <nav class="font-style">
+                <li><a class="toggler-icon"><img class="toggler-img" src="images/humburger.png"
+                            alt="humberger-icon"></a>
+                </li>
```

Comment:
- [ ] Well done 🥳 you did a great job by implementing the changes asked by the previous code reviewer but as I can see that you did extra which is semantically not correct. Kindly follow the below suggestion to not make any mistakes further.
```suggestion
                <img class="toggler-img" src="images/humburger.png"
                            alt="humberger-icon">
```